src/OSPSuite.R/Services/ParameterAnalysableParameterSelector.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.R.Services
+{
+   public class ParameterAnalysableParameterSelector : AbstractParameterAnalysableParameterSelector
+   {
+      public override bool CanUseParameter(IParameter parameter)
```

Comment:
Here is where we filter stuff out. 
So we can never vary parameter that are !CanBeVaried or REadOnlyu or table or categorial (e..g we have the info in core already but hard coded)